pkg/shell/indexes.js

```diff
@@ -614,6 +614,20 @@ function MachinesIndex(index_options, machines, loader, mdialogs) {
 
         if (item && item.docs && item.docs.length > 0)
             item.docs.forEach(e => create_item(_(e.label), e.url));
+
+        // Add 'About Web Console' item
+        const divider = document.createElement("li");
+        divider.className = "divider";
+        const about = document.createElement("li");
+        const el_a = document.createElement("a");
+        el_a.setAttribute("translate", "yes");
```

Comment:
We don't need that bit, and in fact like this it's now actively harmful. The code below already puts in the translated string.

Reply:
palmface

Reply:
Thanks!